python/lab4-propprices.md

```diff
@@ -44,8 +44,9 @@ columns = ['id', 'price', 'date', 'zip', 'type', 'new', 'duration', 'PAON',
            'SAON', 'street', 'locality', 'town', 'district', 'county', 'ppd',
            'status']
 
-rdd = sc.textFile("file:///home/ubuntu/data/prop-prices.csv")
-df = pyspark_csv.csvToDataFrame(sqlContext, rdd, columns=columns)
```

Comment:
This should probably be two separate snippets for Spark 1.x and 2.x -- the `spark.read...csv` method is not going to work in 1.x, AFAICT.